src/RequestState.jsx

```diff
@@ -1,6 +1,31 @@
 // @flow
 
 import {StateFunctorFactory} from 'fronads';
+
+export type RequestState = {
+    emptyFlatMap: (Function) => RequestState,
+    emptyMap: (Function) => RequestState,
+    emptyUnit: (*) => RequestState,
+    errorFlatMap: (Function) => RequestState,
+    errorMap: (Function) => RequestState,
+    errorUnit: (*) => RequestState,
+    fetchingFlatMap: (Function) => RequestState,
+    fetchingMap: (Function) => RequestState,
+    fetchingUnit: (*) => RequestState,
+    refetchingFlatMap: (Function) => RequestState,
+    refetchingMap: RequestState,
```

Comment:
oops, found an error, `refetchingMap` should be `(Function) => RequestState` not `RequestState`